androidshared/src/main/java/org/odk/collect/androidshared/ui/multiclicksafe/MultiClickGuard.kt

```diff
@@ -35,4 +35,14 @@ object MultiClickGuard {
         }
         return allowClick
     }
+
+    /*
+    Utilizing screen names instead of class names offers a more robust approach, as elements placed
+    on the same user interface screen may potentially belong to distinct classes (such as fragments or custom views).
+    Consequently, employing screen names for identification ensures that these elements are recognized
+    as components of the same screen, disabling rapid clicking.
+     */
+    enum class ScreenName {
+        MAIN_MENU
```

Comment:
I like this approach, but I'm thinking this constant should live in the `mainmenu` package? There's no reason for `MultiClickGuard` (or `androidshared`) to know about the concept of the main menu.

Reply:
> There's no reason for MultiClickGuard (or androidshared) to know about the concept of the main menu.

I kind of agree with this but I wanted to keep the list of screens as an enum class so that then when we refactor other classes that use `MultiClickGuard` we can ensure the `allowClick` method is called only with one of those screen names.
Maybe storing that list of screens in the `collect_app` module but not in the `mainmenu` package specifically would be ok?

Reply:
> Maybe storing that list of screens in the collect_app module but not in the mainmenu package specifically would be ok?

Yeah that'd make sense to me!